lib/common.sh

```diff
@@ -296,7 +296,7 @@ run_sbt() {
   local home=$2
   local launcher=$3
   local tasks=$4
-  local buildLogFile=".heroku/sbt-build.log"
+  local buildLogFile=".scalingo/sbt-build.log"
```

Comment:
Sure it doesn't break the build (ie .scalingo directory does not exist?)

Reply:
I updated the JVM common buildpack so that the `.heroku` folder is now named `.scalingo`. So the build failed with `.heroku/sbt-build.log` as the folder does not exist

Reply:
ack